frontend/src/components/MemberTable.vue

```diff
@@ -79,8 +79,10 @@
           </template>
         </div>
       </BBTableCell>
-      <BBTableCell class="tooltip-wrapper">
-        <span class="tooltip">{{ changeRoleTooltip(member) }}</span>
+      <BBTableCell class="whitespace-nowrap tooltip-wrapper">
+        <span v-if="changeRoleTooltip(member)" class="tooltip">
```

Comment:
I'm not an expert in Vue, and wondering if we have better ways to fix it, like syntax in go or swift?
for example,

```
if (let msg = changeRoleTooltip(member)) {
    <span class="tooltip">{{ msg }}</span>
}
```

Reply:
I only know v-if